repo/packages/R/registry/0/config.json

```diff
@@ -0,0 +1,182 @@
+{
+	"$schema": "http://json-schema.org/schema#",
+	"properties": {
+		"service": {
+			"type": "object",
+			"description": "DC/OS service configuration properties",
+			"properties": {
+				"name": {
+					"description": "Name of this service instance",
+					"type": "string",
+					"default": "registry"
+				}
+			}
+		},
+		"registry": {
+			"type": "object",
+			"description": "registry service configuration properties",
+			"properties": {
+				"cpus": {
+					"description": "CPU shares to allocate to each service instance.",
+					"type": "number",
+					"default": 0.2,
+					"minimum": 0.2
+				},
+				"mem": {
+					"description":  "Memory to allocate to each service instance.",
+					"type": "number",
+					"default": 256.0,
+					"minimum": 256.0
+				},
+				"search_backend": {
```

Comment:
what will this be used for? Are there any other supported backends apart from `sqlalchemy`?
should we add this in dcos/examples?